dataikuapi/dss/modelevaluationstore.py

```diff
@@ -288,6 +289,22 @@ def delete(self):
         self.client._perform_json(
                 "DELETE", "/projects/%s/modelevaluationstores/%s/runs/" % (self.project_key, self.mes_id), body=obj)
 
+    def compute_data_drift(self, reference_id=None, data_drift_params=None):
+        """
+        Compute data drift against a reference model or model evaluation. The reference is determined automatically unless specified.
```

Comment:
It seems to me that computing drift against a lab model is not tested. I would make this limitation clearer there.

Reply:
Nothing here specifically mentions "Lab models"? What do you want to change exactly?

(allowing or disallowing this is another debate, to have in the backend PR)

Reply:
Yes it was none mentioned here. I just wanted to make it clear. It seems to me that we decided to have some positive wording. Something like "Compute data drift against a reference saved model version or a model evaluation"